api/defaults.go

```diff
@@ -33,7 +33,8 @@ func defaultSettings(generatedAccountInfo generator.GeneratedAccountInfo, derive
 	s.BackupEnabled = true
 	logLevel := "INFO"
 	s.LogLevel = &logLevel
-	s.ProfilePicturesShowTo = settings.ProfilePicturesShowToContactsOnly
+	s.ProfilePicturesShowTo = settings.ProfilePicturesShowToEveryone
```

Comment:
Thank you for your work @alwx . I do have an objection with the nature of this PR, but not with the implementation and your work. If our messenger app is privacy first I'd argue that we should default to a sensible privacy focused setting, `settings.ProfilePicturesShowToContactsOnly` seems to me to strike that balance.

Otherwise we the concept of `ProfilePicturesShowTo` and `ProfilePicturesVisibility` becomes a bit of a gimmick.

The problem of conforming to other messenger conventions and managing the user's expectations should be an exercise in design and communication, not merely giving the user the loosest privacy levels and hoping they find the privacy toggle.